### PR TITLE
fix: in multi select mode after enter the search is not focused (#265)

### LINKED ISSUE #265: [BUG] multi selection with search does not refocus the search field after selection
**Describe the bug**
If you try out the multiselect example it does not refocus the search field after making the selection, this is bad for keyboard oriented use.

**To Reproduce**
1. Go to https://stackblitz.com/github/bithost-gmbh/ngx-mat-select-search-example?file=src%2Fapp%2Fapp.component.html
2. Go to the example with multiselect
3. Type something in the search
4. Select one item with an enter keypress
5. Try to get back to the search bar --> not possible except using escape and an enter to reopen the search

**Expected behavior**
If pressing escape or some other common key you should be able to refocus the search field to search for another item

### PR DESCRIPTION
**Pull request by bulldog98**
It turns out that a simple fix for the issue is catching the `Enter` keypress and regaining focus after a small timeout.

fixes #265

### CODE REVIEW

**Comment by bulldog98** [comment]
Did I manage to get the commit message format right? If not please feel free to message me I'll fix it after work.

File: src/app/mat-select-search/mat-select-search.component.ts [hunk 1]
```diff
@@ -441,6 +441,11 @@ export class MatSelectSearchComponent implements OnInit, OnDestroy, ControlValue
       event.stopPropagation();
     }
 
+    if (event.key && event.key === 'Enter') {
+      // Regain focus after multiselect, so we can further type
+      setTimeout(() => this._focus(), 200);
```
Comment (macjohnny) [inline src/app/mat-select-search/mat-select-search.component.ts:446]: would it also work without setting a specific delay, i.e. the following?
```suggestion
      setTimeout(() => this._focus());
```

**Comment by macjohnny** [comment]
@bulldog98 thanks for your contribution!

File: src/app/mat-select-search/mat-select-search.component.ts (hunk 1 above)
Comment (bulldog98) [reply src/app/mat-select-search/mat-select-search.component.ts:446]: Yes that seems to work, at least in chromium, I'll try it in firefox.

Comment (bulldog98) [reply src/app/mat-select-search/mat-select-search.component.ts]: Tested it in firefox, work's there, too.

**Comment by macjohnny** [comment]
I will test this a bit more next week and create a new release.

**Comment by bulldog98** [comment]
If you find further problems notify me and I'll try to fix them.

**Review by macjohnny — APPROVED** [review]

**Pull request merged**